src/components/Icon.tsx

```diff
@@ -167,6 +168,7 @@ export enum IconName {
   Website = 'Website',
   Whitepaper = 'Whitepaper',
   Withdraw = 'Withdraw',
+  Download = 'Download',
```

Comment:
nit: can we put alphabetize this in the list (same with below, on line 255?)

Reply:
Done